### PR TITLE
Add todo-count after file name

### PR DESCRIPTION
**Pull request by ghost**
Hello,

I added todo-count after file name in todo list. Feel free to merge this if you think it's good idea.

### CODE REVIEW

File: lib/components/Header.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Controls from './Controls';
 function Header(props){
   return (
     <div className='header'>
-      <h1>todo {
+      <h1>Todo {
```
Comment (reergymerej) [inline lib/components/Header.js]: This was lowercase to match the package name and url (https://atom.io/packages/todo).  It looks like this is consistent everywhere except in the command palette.  Can we lowercase it there instead?

File: lib/components/TreeNode.js [hunk 2]
```diff
@@ -34,7 +38,7 @@ class TreeNode extends React.Component {
         onClick={this.onClick}
         >
         <div className='list-item'>
-          <span className={thisItemClassName}>{this.props.text} ({this.props.nodes.length})</span>
+          <span className={thisItemClassName}>{this.props.text}{childrenCount}</span>
```
Comment (reergymerej) [inline lib/components/TreeNode.js]: Looks like this tweaked the build.
https://travis-ci.org/reergymerej/todo/builds/209111763#L419

**Review by reergymerej — CHANGES_REQUESTED** [review]
@mkoske Thanks for the PR!  I added a couple comments for you to your commits.

**Comment by ghost** [comment]
I'll make new one :)

**Pull request closed without merging**